### DISCUSSION TITLE
System email problems

### DISCUSSION THREAD

**Discussion by JulieVelghe** [verbb/formie#1723] [Q&A]
Hi all, I have a formie form that sends a notification mail.
The recipiënts are the system email variable and another emailadres.
![image](https://github.com/verbb/formie/assets/46524074/2d642e87-21bf-4bd2-9d79-842c26f66457)

in my env file I have the following (x's are placeholders) :
SYSTEM_EMAIL="website@xxxxx.be"
EMAIL_HOSTNAME="smtp-auth.mailprotect.be"
EMAIL_PORT=2525
EMAIL_VERIFICATION=true
EMAIL_USERNAME="website@xxxxx.be"
EMAIL_PASSWORD="xxxxxxxxxx"
EMAIL_ENCRYPTION_METHOD="tls"

And in my craft emailsettings I have: 
![image](https://github.com/verbb/formie/assets/46524074/de003ece-b6c5-45fe-807d-04886418bc15)

Everything worked fine until I updated formie from 2.0.37 to 2.1.2 and Craft from 4.5.6.1 to 4.6.0.
Since then, I'm getting an error that references this system email variable:

Notification email parse error for “To: {systemEmail},xxx@xxx.be”. Template error: “Email "$system_email" does not comply with addr-spec of RFC 2822.” /data/sites/web/xxxxxxx/checkout/main/f73ecfb49b041959d7715531cf72f3e78bc24dec/vendor/symfony/mime/Address.php:54

Does anybody know if this is a bug in Formie or in Craft? Or if this variable has been changed?

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.3"`.
